python/tk_rv_shotgunreview/rv_activity_mode.py

```diff
@@ -1007,10 +1007,10 @@ def __init__(self, app):
                 ('launch-submit-tool', self.launch_submit_tool, ''),
                 ('submit-tool-submission-complete', self.version_submitted, ''),
                 # Shortcuts:
-                ('key-down--alt--d', self.toggle_view_details, ''),
-                ('key-down--alt--t', self.toggle_view_tray, ''),
-                ('key-down--alt--c', self.cuts_button_pushed_event, ''),
-                ('key-down--alt--m', self.mini_cut_mode_toggle, ''),
+                ('key-down--alt--D', self.toggle_view_details, ''),
+                ('key-down--alt--T', self.toggle_view_tray, ''),
+                #('key-down--alt--c', self.cuts_button_pushed_event, ''),
+                #('key-down--alt--m', self.mini_cut_mode_toggle, ''),
```

Comment:
why are we taking these two out?

Reply:
We have alt+c already used to copy the path of the current source to the clipboard, but I guess I could have kept alt+m